### DISCUSSION TITLE
Stable Diffusion - need help to enable xformers (alternative to: --skip-torch-cuda-test ---precision full --no-half)

### DISCUSSION THREAD

**Discussion by Wik9664** [TheLastBen/fast-stable-diffusion#2791] [Q&A]
xformers doesn't seem to work on one of my computers, so I've been running SD on A1111 with the following commands:

--autolaunch --medvram --skip-torch-cuda-test --precision full --no-half
No module 'xformers'. Proceeding without it. 
Warning: caught exception 'Found no NVIDIA driver on your system'...

This enables SD to function, but its pretty slow compared to my other computer, which can use xformers. I assume the problem might be the lack of recognition for the NVIDIA driver, but I'm not sure how to troubleshoot that. 

Is there a simple way of getting xformers to work, without this second-best alternative: '--skip-torch-cuda', etc?

Thanks for any suggestions!

**Comment by TheLastBen** [comment]
use `--opt-sdp-attention`, it's as good as --xformers

**Comment by Wik9664** [comment]
Thanks. I think I've tried that in the past (seems familiar), but I'll give it another go.

Not sure if I'll need to keep '--skip-torch-cuda-test --precision full --no-half' as well, but I'll try it with and without and see what works.
